tests/dataset_tests/parsers_tests/test_sdf_file_parser.py

```diff
@@ -27,6 +27,19 @@ def sdf_file(tmpdir, mols):
     return fname
 
 
+@pytest.fixture()
+def sdf_file_long(tmpdir):
+    """SDFFile with long smiles (ccc...)"""
+    fname = os.path.join(str(tmpdir), 'test_long.sdf')
+    writer = Chem.SDWriter(fname)
+    for smiles in ['CCCCCCCCCCCC', 'CN=C=O', 'CCCCCCCCCCCCCCCC',
+                   'Cc1ccccc1', 'CC1=CC2CC(CC1)O2']:
+        mol = Chem.MolFromSmiles(smiles)
+        writer.write(mol)
```

Comment:
I dont know how to create "invalid" sdf file.
So I made sdf file with larger size of smiles (more than 10 atoms).

Reply:
If you don't want to make invalid 'sdf', this code can make it.(1st mol is broken.)

```
with open('test_long.sdf', 'r') as f:
    lines = f.readlines()
del(lines[10])
with open('test_long.sdf', 'w') as f:
    f.writelines(lines)
```

Reply:
I see thanks. but removing middle of the line (1st mol within total 3 mols etc.)  is difficult, so may i proceed for current PR?

Reply:
OK.